scripts/create-release-archive.js

```diff
@@ -0,0 +1,193 @@
+const child_process = require('child_process') // eslint-disable-line camelcase
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const packageJsonScriptsInclude = [
+  'start',
+  'build',
+  'serve'
+]
+
+function usage () {
+  console.log(`
+Usage:
+    scripts/create-release-archive [options] [REF]
+
+Options
+    -h, --help
+
+Options (advanced):
+    --destDir DIR
+    --releaseVersion VERSION
```

Comment:
Should --releaseVersion be --releaseName in the console.log?

Reply:
Fixed, thanks